delocate/delocating.py

```diff
@@ -133,10 +134,23 @@ def _sanitize_rpaths(
     files_to_delocate: Iterable[str],
 ) -> None:
     """Sanitize the rpaths of libraries."""
+    requiring_sanitizes = {}  # requiring -> True
+
     for required in files_to_delocate:
         # Set relative path for local library
         for requiring, orig_install_name in lib_dict[required].items():
+            requiring_sanitizes[requiring] = True
+
+    try:
+        from joblib import Parallel, cpu_count, delayed
```

Comment:
`joblib` is untyped and must be explicitly marked as such:
```suggestion
        from joblib import Parallel, cpu_count, delayed  # type: ignore[import-untyped]
```